### PR TITLE
Fix a leftover System::log call

### LINKED ISSUE #3948: Systemwartung: Fehler bei Job "Vorschau-Cache leeren"
### Affected version(s)

4.13.0-RC1

### Description

Die Ausführung der Systemwartung im Backend führt beim Job "Vorschau-Cache leeren" zu folgendem Fehler:

```
Internal Server Error
Was ist das Problem?
Attempted to load class "ContaoContext" from namespace "Contao". Did you forget a "use" statement for e.g. "Contao\CoreBundle\Asset\ContaoContext" or "Contao\CoreBundle\Monolog\ContaoContext"?
```

Im error.log des Servers sehe ich folgende Meldung:

`2022/01/18 12:13:20 [error] 1112#1112: *12626 FastCGI sent in stderr: "PHP message: PHP Warning:  scandir(/var/www/clients/web/assets/previews/): Failed to open directory: No such file or directory in /var/www/clients/web/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Folder.php on line 586PHP message: PHP Warning:  scandir(): (errno 2): No such file or directory in /var/www/clients/web/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Folder.php on line 586PHP message: PHP Warning:  foreach() argument must be of type array|object, bool given in /var/www/clients/web/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Folder.php on line 586" while reading response header from upstream, client: 111.111.111.111, server: abc.xyz, request: "POST /contao?do=maintenance&ref=My5VwcaP HTTP/1.1", upstream: "fastcgi://unix:/var/lib/php5-fpm/web10.sock:", host: "abc.xyz", referrer: "http://abc.xyz/"`

### PR DESCRIPTION
**Pull request by fritzmg**
fixes #3948

This also removes some remaining `System::log` calls as mentioned by https://github.com/contao/contao/pull/3949#issuecomment-1015347638

/cc @ausi @SeverinGloeckle

### CODE REVIEW

**Review by m-vo — DISMISSED** [review]

**Comment by fritzmg** [comment]
Failing test is unrelated.

**Comment by SeverinGloeckle** [comment]
Good catch.

On a quick search, I found more `System::log` calls (from https://github.com/contao/contao/commit/fa85278a4829a06c3076578504edce8790c19133). These do not trigger exceptions (since they do not use `ContaoContext`), but maybe you can replace them as well to get rid of the deprecations:

https://github.com/contao/contao/blob/4b4d6adb7dd9ff76f095142a7e6f9fcc0ba7d7e3/core-bundle/src/Resources/contao/modules/ModuleQuicklink.php#L135-L141

https://github.com/contao/contao/blob/4b4d6adb7dd9ff76f095142a7e6f9fcc0ba7d7e3/core-bundle/src/Resources/contao/modules/ModuleCustomnav.php#L141-L147

https://github.com/contao/contao/blob/4b4d6adb7dd9ff76f095142a7e6f9fcc0ba7d7e3/core-bundle/src/Resources/contao/modules/ModuleQuicknav.php#L151-L156

File: core-bundle/src/Resources/contao/library/Contao/Automator.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ public function purgePreviewCache()
 		$this->purgePageCache();
 
 		// Add a log entry
```
Comment (SeverinGloeckle) [inline core-bundle/src/Resources/contao/library/Contao/Automator.php]:
```suggestion
```

I removed the redundant `//Add a log entry` comments when replacing the log calls initially. I'd suggest removing this comment as well.

**Review by m-vo — APPROVED** [review]

**Review by bytehead — APPROVED** [review]

**Review by ausi — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**